lib/src/util/react_wrappers.dart

```diff
@@ -96,8 +111,8 @@ final bool _canUseExpandoOnReactElement = (() {
 ///
 /// If caching isn't possible due to [_canUseExpandoOnReactElement] being false,
 /// then this will be initialized to `null`, and caching will be disabled.
-final Expando<UnmodifiableMapView> _elementPropsCache = _canUseExpandoOnReactElement
-    ? new Expando<UnmodifiableMapView>('_elementPropsCache')
+final Expando<Map> _elementPropsCache = _canUseExpandoOnReactElement
+    ? new Expando<Map>('_elementPropsCache')
```

Comment:
What is the `tldr;` on the type change here?

Reply:
The `UnmodifiableMapView` typing is unnecessarily specific, and in the future it's possible that these could be (frozen) `JsMap` instances. I went down the path of trying that out but couldn't. See comment below:
> ```dart
> // TODO Since JS props are frozen don't wrap in UnmodifiableMapView once https://github.com/dart-lang/sdk/issues/15432 is fixed
> rawPropsOrCopy = JsBackedMap.backedBy(instance.props);
> ```